test/e2e/perf_util.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/kubernetes/test/e2e/perftype"
+)
+
+// apiCallToPerfData transforms APIResponsiveness to PerfData. This is used by Density test
+// and Load test now.
+func apiCallToPerfData(apicalls APIResponsiveness) *perftype.PerfData {
+	perfData := &perftype.PerfData{}
+	for _, apicall := range apicalls.APICalls {
+		item := perftype.DataItem{
+			Data: map[string]float64{
+				"Perc50": float64(apicall.Latency.Perc50) / 1000000, // us -> ms
+				"Perc90": float64(apicall.Latency.Perc90) / 1000000,
+				"Perc99": float64(apicall.Latency.Perc99) / 1000000,
+			},
+			Unit: "ms",
+			Tags: map[string]string{
+				"Verb":     apicall.Verb,
+				"Resource": apicall.Resource,
+			},
+		}
+		perfData.DataItems = append(perfData.DataItems, item)
+	}
+	return perfData
+}
```

Comment:
BTW - we should probably change the tests to actually use perftype.PerfData from the begining instead of translating one to the other here.
Can you please add a TODO?